site/content/api/Device-Connection-API.md

```diff
@@ -0,0 +1,148 @@
+| *400* | Bad Request, the last known gateway has NOT been updated due to invalid data in the request. |
+
+Implementors of this API may return a *404* status code in order to indicate that no device and/or gateway with the given identifier exists for the given tenant. However, performing such a check is optional.
+
+For status codes indicating an error (codes in the `400 - 499` range) the message body MAY contain a detailed description of the error that occurred.
+
+### Get Last Known Gateway for Device
+
+Clients use this command to *retrieve* the gateway that last acted on behalf of a given device.
+
+**Message Flow**
+
+
+The following sequence diagram illustrates the flow of messages involved in a *Client* retrieving the last known gateway for a Device.
+
+![Get Last Known Gateway for Device message flow](../get-last-gw_Success.png)
+
+
+**Request Message Format**
+
+The following table provides an overview of the properties a client needs to set on a message to retrieve the last known gateway for a device in addition to the [Standard Request Properties]({{< relref "#standard-request-properties" >}}).
+
+| Name        | Mandatory | Location                 | AMQP Type | Description |
+| :---------- | :-------: | :----------------------- | :-------- | :---------- |
+| *subject*   | yes       | *properties*             | *string*  | MUST be set to `get-last-gw`. |
+
+The body of the message SHOULD be empty and will be ignored if it is not.
+
+**Response Message Format**
+
+A response to a *get last known gateway for device* request contains the [Standard Response Properties]({{< relref "#standard-response-properties" >}}) as well as the properties shown in the following table:
+
+| Name             | Mandatory | Location                 | AMQP Type    | Description |
+| :--------------- | :-------: | :----------------------- | :----------- | :---------- |
+| *content-type*   | no        | *properties*             | *string*     | MUST be set to `application/json` if the invocation of the operation was successful and the body of the response message contains payload as described below. |
+
+The result of a successful invocation is carried in a single Data section of the response message as a UTF-8 encoded string representation of a single JSON object. It is an error to include payload that is not of this type.
+
+The response message JSON object has the following properties:
+
+| Name             | Mandatory | JSON Type     | Description |
+| :--------------- | :-------: | :------------ | :---------- |
+| *gateway-id*     | *yes*     | *string*      | The ID of the last known gateway for the device. |
+| *update-date*    | *no*      | *string*      | The date that the information about the last known gateway for the device was last updated. The value MUST be an [ISO 8601 compliant *combined date and time representation in extended format*](https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations). |
```

Comment:
*update-date* sounds a little weird, doesn't it? What about just *updated* or *last-updated* instead?

Reply:
yes, I've changed it to `last-updated`.